Cargo.toml

```diff
@@ -50,3 +50,9 @@ git = "https://github.com/launchbadge/sqlx/"
 branch = "master"
 default-features = false
 features = ["runtime-actix", "postgres", "chrono", "offline"]
+
+
+[dependencies.actix-ratelimit]
+# Temp fork until actix-ratelimit bumps version on cargo
+git = "https://github.com/TerminalWitchcraft/actix-ratelimit"
+branch = "master"
```

Comment:
1. This isn't a fork of `actix-ratelimit`, that's the official repo
2.  This should specify the commit hash using `rev` instead of setting the branch to `master` so that it doesn't break in the future.
    - In this case it would be `rev = "870822067dfeae7cc0304352d81c4cb79ee27f5a"`